frontend/server/src/Controllers/Problem.php

```diff
@@ -3816,13 +3816,51 @@ public static function getProblemEditDetailsForSmarty(
             $r,
             /*$isRequired=*/ false
         );
-
+        $validatorsTypes = [
+            \OmegaUp\ProblemParams::VALIDATOR_TOKEN_CASELESS => \OmegaUp\Translations::getInstance()->get(
+                'problemEditFormTokenCaseless'
+            ),
+            \OmegaUp\ProblemParams::VALIDATOR_TOKEN_NUMERIC => \OmegaUp\Translations::getInstance()->get(
+                'problemEditFormNumericTokensWithTolerance'
+            ),
+            \OmegaUp\ProblemParams::VALIDATOR_TOKEN => \OmegaUp\Translations::getInstance()->get(
+                'problemEditFormTokenByToken'
+            ),
+            \OmegaUp\ProblemParams::VALIDATOR_LITERAL => \OmegaUp\Translations::getInstance()->get(
+                'problemEditFormLiteral'
+            ),
+            \OmegaUp\ProblemParams::VALIDATOR_CUSTOM => \OmegaUp\Translations::getInstance()->get(
+                'problemEditFormCustom'
+            )
+        ];
+        $sortedLanguages = \OmegaUp\Controllers\Run::DEFAULT_LANGUAGES;
+        sort($sortedLanguages);
+        $validLanguages = [ join(
```

Comment:
nit:

```suggestion
        $validLanguages = [
            join(
                ....
            ) => 'C....',
            'kj,kp' => 'Karel',
            ...,
        ];    
```

los elementos de los arreglos deben ir en una sóla línea, o un elemento por línea.

Reply:
listo